pygments/cmdline.py

```diff
@@ -330,24 +229,21 @@ def main_inner(popts, args, usage):
         return 0
 
     # if no -S is given, -a is not allowed
-    if a_opt is not None:
```

Comment:
Can this be moved up to line 211, right after `S_opt` and `a_opt` is read from `argsn`? It looks like you'd want to handle this early and not somewhere in the middle.

Reply:
no, otherwise `-a` would never be allowed. The point is to disallow `-a` if `-S` is not present.